lib/services/servers_test.go

```diff
@@ -111,27 +112,32 @@ func (s *ServerSuite) TestServersCompare(c *check.C) {
 	c.Assert(CompareServers(node, &node2), check.Equals, Different)
 }
 
+// TestGuessProxyHostAndVersion checks that the GuessProxyHostAndVersion
+// correctly guesses the public address of the proxy (Teleport Cluster).
 func (s *ServerSuite) TestGuessProxyHostAndVersion(c *check.C) {
-	// nil proxies
-	host, version := GuessProxyHostAndVersion(nil)
+	// No proxies passed in.
+	host, version, err := GuessProxyHostAndVersion(nil)
 	c.Assert(host, check.Equals, "")
 	c.Assert(version, check.Equals, "")
+	c.Assert(err, check.FitsTypeOf, trace.NotFound(""))
```

Comment:
how about: `c.Assert(trace.IsNotFound(err), check.Equals, true)`?

Reply:
The `fixtures.ExpectNotFound` function exists for this purpose.